test/extension/io/png_read_test.cpp

```diff
@@ -728,4 +728,42 @@ BOOST_AUTO_TEST_CASE( gamma_test )
 
 #endif // BOOST_GIL_IO_USE_PNG_TEST_SUITE_IMAGES
 
+
+BOOST_AUTO_TEST_CASE( corrupted_png_read_test ) {
+// taken from https://github.com/boostorg/gil/issues/401#issue-518615480
+// author: https://github.com/misos1
+
+	std::initializer_list<unsigned char> corrupt_png = {
```

Comment:
Please, indent with 4 spaces and no tabs.

https://github.com/boostorg/gil/blob/develop/CONTRIBUTING.md#guidelines